doc/sphinx/source/recipes/recipe_hyint.rst

```diff
@@ -0,0 +1,128 @@
+
+The hydroclimatic indices calculated by the diagnostic and included in the output are defined as follows:
+
+* PRY = mean annual precipitation
+* INT = mean annual precipitation intensity (intensity during wet days, or simple precipitation intensity index SDII)
+* WSL = mean annual wet spell length (number of consecutive days during each wet spell)
+* DSL = mean annual dry spell lenght (number of consecutive days during each dry spell)
+* PA  = precipitation area (area over which of any given day precipitation occurs)
+* R95 = heavy precipitation index (percent of total precipitation above the 95% percentile of the reference distribution)
+* HY-INT = hydroclimatic intensity. HY-INT = normalized(INT) x normalized(DSL).
+ 
+ 
+ 
+Available recipes and diagnostics
+---------------------------------
+ 
+Recipes are stored in recipes/
+ 
+* recipe_hyint.yml (evaluating the 6 hydroclimatic indices) 
+* recipe_hyintex.yml (also including a selection of ETCCDI indices)
```

Comment:
This recipe does not appear to be included in this pull request. Can you add it?

Reply:
The second recipe is on hold because of the delay in porting extremeEvents in v2. Hyint reads the results from extremeEvents. Removed "recipe_hyintex.yml" line from doc.